dig_test.go

```diff
@@ -727,6 +876,21 @@ func TestInvokeFailures(t *testing.T) {
 		require.Contains(t, err.Error(), "dig.type2 isn't in the container")
 	})
 
+	t.Run("unmet named dependency", func(t *testing.T) {
```

Comment:
That is not supported yet. See the TODO about an index.
I would prefer to defer that to a new PR